app/views/devise/registrations/_registration_form.html.erb

```diff
@@ -22,20 +30,35 @@
       <%= render partial: "shared/authentication/providers_registration_form" %>
 
       <% if params[:state] == "new-user" %>
-        <div class="registration__hr">
-          <span class="registration__hr-label">
-            Already have an account? <a href="<%= sign_up_path %>">Log in</a>.
-          </span>
+
+        <div class="registration__agreement-text">
+          By signing up, you are agreeing to our <a href="/privacy">privacy policy</a>, <a href="/terms">terms of use</a> and <a href="/code-of-conduct">code of conduct</a>.
```

Comment:
We should use the path tags here instead of the urls. i.e. privacy_path, code_of_conduct_path, etc.

Reply:
Applied this change. Thanks.